.env

```diff
@@ -0,0 +1,8 @@
+API_KEY="eEvdAEQQzFedHQs5h0H04qevA"
+API_SECRET="AFPPyICgVQHvhudG68N4ak8uWwUerV4pEHPgRfjz3BeX4hPloR"
+ACCESS_TOKEN="4855079537-7whIoJLZTdYV7yvap79mv8vif83RsJqgZ3B9gFi"
+ACCESS_TOKEN_SECRET="W8dg9Ft2lgZzITApLZ2sZhuaXC3X3YfIfgchkA59L3uBp"
+GOOGLE_MAP_API="x"
+WALKSCORE_API="361ea4b83fe597e92ba529fa750027e4"
+INDICO_API="0c9fa38eaa878478f49be9cb71c3d0a7"
+JAWSDB_URL="mysql://wdjh17ha3wgo5702:ujp8eauf6080ov4k@icopoghru9oezxh8.cbetxkdyhwsb.us-east-1.rds.amazonaws.com:3306/xzrctothavjrrohs"
```

Comment:
I see this is new, but is there a way to remove the other .env file info so that it isn't floating around still?